firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -588,50 +593,40 @@ void update_brake_pressure( )
 // Returns:     void
 // 
 // Parameters:  uint8_t sensor_p_left
-//				uint8_t sensor_p_right
-//				uint8_t solenoid_pin_left_a
-//				uint8_t solenoid_pin_right_a
-//				uint8_t solenoid_pin_left_r
-//				uint8_t solenoid_pin_right_r
+//              uint8_t sensor_p_right
+//              uint8_t solenoid_pin_left_a
+//              uint8_t solenoid_pin_right_a
+//              uint8_t solenoid_pin_left_r
+//              uint8_t solenoid_pin_right_r
 // 
 // *****************************************************
 void init_brakes( uint8_t sensor_p_left,
```

Comment:
Does it make sense to create a local brake structure and pass that in to this one?

Use the existing one and initialize it with a { 0, 0, 0, 0, ....}; and then pass in a pointer to the locally declared structure when init_brakes is called.  I think MISRA doesn't like any more than two parameters, maybe three.